sherpa/astro/datastack/ds.py

```diff
@@ -28,6 +27,10 @@
 
 logger = config_logger(__name__)
 
+try:
+    import stk
+except:
+    logger.warning("could not import stk library. CIAO stack files and syntax will be disabled")
```

Comment:
Is the idea here that, because the only use of the `stk` module is already within a try/except block, there is no need to create a "replacement" routine for `stk.build()` in the except clause? I thought that the change looked odd, until I checked into how the `stk` module was used here. Do we need to document this (e.g. in a comment) for anyone who wants to use `stk,build` in this module? Or am I being over sensitive?